deploy/internal/statefulset-postgres-db.yaml

```diff
@@ -26,6 +26,8 @@ spec:
         - name: db
           image: NOOBAA_DB_IMAGE
           env:
+            - name: POSTGRESQL_UPGRADE
```

Comment:
This is only relevant for upgrades. when hardcoded here it will set the env for every new deployment as well.
The `POSTGRES_UPGRADE` env should be set only when necessary

Reply:
Yes. This is not needed anymore. The reconcile will detect upgrade status and flag will be set on the fly in memory